opensrp-reveal/src/main/java/org/smartregister/reveal/model/OfflineMapModel.java

```diff
@@ -0,0 +1,56 @@
+package org.smartregister.reveal.model;
+
+import com.mapbox.mapboxsdk.offline.OfflineRegion;
+
+import org.smartregister.domain.Location;
+
+public class OfflineMapModel {
+
+    private Location location;
+
+    private OfflineRegion offlineRegion;
+
+    public enum OFFLINE_MAP_STATUS {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The enum name 'OFFLINE_MAP_STATUS' doesn't match '[A-Z][a-zA-Z0-9]*'](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=4217760)